zokrates_core_test/tests/tests/assembly/bitify.zok

```diff
@@ -0,0 +1,21 @@
+def bitify<N>(field num) -> field[N] {
+    field[N] mut out = [0; N];
+    field mut lc1 = 0;
+    field mut e2 = 1;
+    for u32 i in 0..N {
+        asm {
+            out[i] <-- (num >> i) & 1;
```

Comment:
This is using little-endian, while the rest of the codebase uses big-endian. I think it would make sense to have consistency and use big-endian everywhere.